component_sdk/python/kfp_component/google/ml_engine/_create_version.py

```diff
@@ -25,13 +26,12 @@
 from ._common_ops import wait_existing_version, wait_for_operation_done
 
 @decorators.SetParseFns(python_version=str, runtime_version=str)
-def create_version(project_id, model_name, deployemnt_uri=None, version_name=None, 
```

Comment:
Why expose the model_name and parse the model name? how about keep the project_id and construct the model_name inside the op?

Reply:
Discussed offline. Rename short names to be name or model_short_name and changed comment to be reflect that.

Reply:
Discussed offline added. change the user provided name to id, for example model_id, project_id, version_id. The returned is model_name, version_name.